### DISCUSSION TITLE
Add FullCalendar options for calendar layout

### DISCUSSION THREAD

**Discussion by YannickMol** [directus/directus#5842] [Feature Requests]
It would be nice to add FullCalendar options to be able to change some of the standard elements. An example would be the day the week begins (https://fullcalendar.io/docs/firstDay).

**Comment by rijkvanzanten** [comment]
_Iedereen weet toch dat de week op maandag begint pff_

**Comment by abdonrd** [comment]
I'm also interested in the week starting on Monday! 😄

**Reply by rijkvanzanten** [reply]
Wanna open a PR to add that as a layout option? 🙂 Unlike adding the second line in the event preview, this should be trivial to add, as fullCalendar already supports it out of the box 😄

**Reply by abdonrd** [reply]
Seems like it take care about the locales:
https://fullcalendar.io/docs/firstDay

**Reply by rijkvanzanten** [reply]
> Seems like it take care about the locales:
> https://fullcalendar.io/docs/firstDay

I still think it makes sense to expose that option as a setting in the layout options in Directus. I use `en-US` as my locale, but still prefer weeks to start on Mondays

**Reply by abdonrd** [reply]
Yeah, the same happens to me!

**Reply by hintva** [reply]
Hi ! Did you open a PR on that subject finally ? We could be interested in participating, or developing an extension layout for calendar, including a "resources" display, such as this layout on FullCalendar : https://fullcalendar.io/demos > Resource Timeline. 
If something is ongoing we could join, or at least not make the work twice. Thank you !

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
